test/unit/web-runtime/next-response.test.ts

```diff
@@ -53,3 +12,8 @@ it('can be cloned', async () => {
     server: 'Vercel',
   })
 })
+
+it('can return JSON', async () => {
```

Comment:
Should this test work since we don't allow returning response bodies now?

Reply:
Yes, because this is still something that can be used in Edge API endpoints

Reply:
Ah ok misunderstand when removing these then 258e44796fbc26c4fe371af754af77a315ecff12 will re-add but seems we need to fix the type as it doesn't seem to expect `Response.json` to be available